src/client/pages/BffError/BffError.tsx

```diff
@@ -18,7 +23,11 @@ export default function Bff500Error() {
         <p>
           Er is een fout opgetreden in de communicatie met de server.{' '}
           <LinkdInline href="/">Ga verder naar home.</LinkdInline>
-          {!IS_PRODUCTION ? <pre>{stack}</pre> : ''}
+          {!IS_PRODUCTION ? (
```

Comment:
is dit niet mooier?:{!IS_PRODUCTION && <pre style={{ whiteSpace: 'break-spaces' }}>{stack}</pre>}

Reply:
ja!